amundsen_application/static/config/config-default.ts

```diff
@@ -34,7 +21,22 @@ const configDefault: AppConfig = {
       href: "/browse",
       use_router: true,
     }
-  ]
+  ],
+  tableLineage: {
+    iconPath: 'PATH_TO_ICON',
+    isBeta: false,
+    isEnabled: false,
+    urlGenerator: (database: string, cluster: string, schema: string, table: string) => {
+      return `https://DEFAULT_LINEAGE_URL?schema=${schema}&cluster=${cluster}&db=${database}&table=${table}`;
+    },
+  },
+  tableProfile: {
+    isBeta: false,
+    isExploreEnabled: false,
+    exploreUrlGenerator: (database: string, cluster: string, schema: string, table: string, partitionKey?: string, partitionValue?: string) => {
+      return `https://DEFAULT_EXPLORE_URL?schema=${schema}&cluster=${cluster}&db=${database}&table=${table}`;
+    }
+  },
```

Comment:
Any configs related to the Data Preview button should also fall under `tableProfile` if and when we add them.

Reply:
the only concern is that external user may have confused what it means by isBeta(not sure if everyone ring a bell for beta, GA term). Otherwise LGTM

Reply:
There are some comments about these configs in config.types.ts.